supplementary_style_guide/style_guidelines/statements-about-the-future.adoc

```diff
@@ -0,0 +1,25 @@
+[[statements-about-the-future]]
+= Statements about the future
+
+[[making-statements-about-future-releases-or-plans]]
+== Making statements about future releases or plans
+
+Regarding future product releases, the _IBM Style Guide_ states, "Avoid claims that are related to the content or timing of a future product or release."  
```

Comment:
I don't think that we'd want to state ISG's guidelines word for word. I'd try to reword. Maybe something like

```
When possible, avoid making statements that predict future releases or plans.
```